channel.go

```diff
@@ -228,23 +268,51 @@ func (c *serialChannel) teardown() error {
 	return c.serialConn.Close()
 }
 
+// isAFVSockSupported checks if vsock channel is used by the runtime
+// by checking for devices under the vhost-vsock driver path.
+// It returns true if more a device is found for the driver.
```

Comment:
nit: can you clarify this last sentence?

Reply:
done